### ISSUE TITLE
Neither default `READ_PAIR_CONCORDANT_PERCENT` or `--concordantreadpairdistribution` used in `gridss.sh`

### ISSUE THREAD

**Issue by alexiswl** [PapenfussLab/gridss#444]
`readpairpdistribution` is initialised as `0.995` here: 
https://github.com/PapenfussLab/gridss/blob/cf23276f519d3a305223dca2109e57457dde9ad1/scripts/gridss.sh#L42

<br></br>

This can then be overridden in the arguments here using the `--concordantreadpairdistribution` parameter:
https://github.com/PapenfussLab/gridss/blob/cf23276f519d3a305223dca2109e57457dde9ad1/scripts/gridss.sh#L146-L149

<br></br>

`readpairing_args` is initialised as `null` here:
https://github.com/PapenfussLab/gridss/blob/cf23276f519d3a305223dca2109e57457dde9ad1/scripts/gridss.sh#L518

<br></br>

But there is no redefining `readpairing_args` with the `readpairpdistribution` value:
https://github.com/PapenfussLab/gridss/blob/cf23276f519d3a305223dca2109e57457dde9ad1/scripts/gridss.sh#L520-L523

<br></br>

In conclusion: `readpairing_args` is not affected by the `--concordantreadpairdistribution` parameter or the default `readpairpdistribution` value in anyway.  It is set to `READ_PAIR_CONCORDANT_PERCENT=null` if `--useproperpair` is set to `"true"`, otherwise it remains as it's initial value: `""`
